encrypted-media-respec.html

```diff
@@ -609,19 +609,24 @@
               <p>
               Supported configurations, including supported audio and video codecs, may depend on availability of optional capabilities such as
               <a def-id="distinctive-identifier">Distinctive Identifier(s)</a> and persistent state. The following algorithm iteratively tries to find a configuration
-              that is both supported and has user consent, each iteration considering a smaller set of possible configurations.
+              that is both supported and has user consent (or does not need consent).
               </p>
               <p>
               User Agents should reuse earlier consent responses, when appropriate, at least for the duration of the <a def-id="requestMediaKeySystemAccess"></a>
               algorithm in order to avoid repeated requests to the user for similar configurations.
               </p>
+              <p>
+                The variable <var>restrictions</var> in the steps below represents the configurations for which consent has been denied. It is used to determine
+                whether user consent for a candidate configuration or partial configuration has been denied. Consent is denied for a partial configuration if every derived
```

Comment:
Perhaps s/has been denied/would be denied based on previous {information obtained in previous iterations}/

The text in {} may need improvement, but the important part is past=>future for _this_ configuration.